starter-code/src/functions-and-arrays.js

```diff
@@ -10,14 +17,56 @@ var words = [
   'orchard',
   'crackpot'
 ];
+ function findLongestWord(array) {
+  var word = [];
+  for (var i = 0; i < array.length; i++) {
+    if (word.length < array[i].length) {
+      word = array[i];
+    }
+    
+  }
+  if(array.length===0){
+    return undefined
+  }
+  return word;
+ }
+ findLongestWord(words);
+ 
+
+
 
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+function sumArray ( numbers ) {
+  
+  var sum =0;
+  for(var i=0;i<numbers.length;i++){
+    sum= sum + numbers[i];
```

Comment:
Perfect!

You can also write :     sum= sum + numbers[i];
this way : sum +=  numbers[i];